src/install/install.zig

```diff
@@ -1910,6 +1910,7 @@ const Waiter = struct {
 // 2.
 pub const PackageManager = struct {
     cache_directory_: ?std.fs.Dir = null,
+    cache_directory_path: string = "",
```

Comment:
Please add a comment to remove this once we move away from using std.ChildProccess for `git clone`.